data/publiccloud/terraform/azure.tf

```diff
@@ -182,9 +182,11 @@ resource "azurerm_linux_virtual_machine" "openqa-vm" {
   }
 
   os_disk {
-    name              = "${var.name}-${element(random_id.service.*.hex, count.index)}-osdisk"
+    name                 = "${var.name}-${element(random_id.service.*.hex, count.index)}-osdisk"
     caching              = "ReadWrite"
     storage_account_type = "Standard_LRS"
+    # SLE images are 30G by default 
```

Comment:
I'm pretty sure we will not understand this comment in a year from now so I'd remove it. Why?

If I would see this comment, then I would ask the question: "if those are 30G by default, why do we explicitly state `disk_size_gb = 30` in the next line?"
We now know why this is there, but without this knowledge this comment might be confusing.

Reply:
The comment now says:
```
# SLE images are 30G by default so the next line is currently unnecessary
```

Reply:
I still think it's confusing..  you say that the line is unnecessary but it is still there :D 
I would simple remove it or say `size can be >=30G`

Reply:
How about

    # The default for SLE images is currently 30G. The following line is kept here in case we need to increase the disk size

Reply:
if we speaking about "documentation commit" let's keep it like this . 

```
# SLE images are 30G by default. Uncomment this line in case we need to increase the disk size ( note: value can not  be decreased because 30 GB is minimum allowed by Azure ) 
# disk_size_gb         = 30
```